clients/privacy-center/public/test/fides-js-components-demo.html

```diff
@@ -0,0 +1,234 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>fides-js script demo page</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <!-- 
+      We use the fides*.js file directly so that we can dynamically inject options.
+      We read fidesConfig.options to determine which fides*.js bundle to use.
+    -->
+    <script>
+      (function () {
+        var src = window.fidesConfig?.options?.tcfEnabled
+          ? "../lib/fides-tcf.js"
+          : "../lib/fides.js";
```

Comment:
OK - this works now, but I think this test page is just creating tech debt in the repo. I'm going to delete this and merge 👍